eventbridge-bedrock-s3-aoss/example-pattern.json

```diff
@@ -0,0 +1,100 @@
+{
+  "title": "# EventBridge to Amazon Bedrock to Amazon OpenSearch Serverless",
+  "description": "Periodically sync Bedrock Knowledge Base",
```

Comment:
Pls elaborate the description a bit but under 150 char.

Reply:
done